src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -18,26 +23,25 @@ const defaultFormValues = { create: '', searchExisting: '' };
   templateUrl: './add-content.component.html',
   styleUrls: [ './add-content.component.scss' ],
 })
-export class AddContentComponent implements OnInit, OnDestroy {
-  @Input() allowSkills = false;
-  @Output() contentAdded = new EventEmitter<AddedItem>();
+export class AddContentComponent<Type, SearchedValue extends AddedContent<Type>> implements OnInit, OnDestroy {
+
+  @Input() newContentTypes: NewContentType<Type>[] = [];
+  @Input() foundValues: SearchedValue[] = [];
```

Comment:
Not clear what it is exactly for. To clarify, you should call it "resultsFromSearch".

Reply:
fixed